src/Nokia_LCD.cpp

```diff
@@ -107,6 +115,15 @@ void Nokia_LCD::setContrast(uint8_t contrast) {
 
 void Nokia_LCD::setInverted(bool invert) { mInverted = invert; }
 
+void Nokia_LCD::setFont(CustomFont *font) { 
+    if (font != NULL) 
+        customFont = font;         
+    else
+        customFont = &defaultFont;
```

Comment:
Considering the previous comment about setting the font in the constructor's initializer list, this should be turned into:

```suggestion
    if (!font) {
        return;
    }
    customFont = &defaultFont;
```

Reply:
I got the idea, but just one point here, the last line (122) should be `customFont = font;`, right?

Reply:
Yeap